pkg/autoupdate/autoupdate.go

```diff
@@ -174,6 +183,36 @@ func AutoUpdate(runtime *libpod.Runtime, options Options) ([]string, []error) {
 		}
 	}
 
+	// Autoupdate Local Iteration
+	for imageID, containers := range localMap {
```

Comment:
That adds quite some duplicate code.  I think we should have only loop and make the `needsUpdate` check conditional on the policy.  We could achieve that by merging the maps into one.

`map[imageUpdate][]*libpod.Container` with
```golang
type imageUpdate struct {
    image string
    policy string
}
```

That allows for bundling an image with a policy:
```golang
for imageUpdate, containers := range myMap {
    switch imageUpdate.policy {
    case ...:
    case ...:
    }
}
```